AUTOMATION.md

```diff
@@ -0,0 +1,40 @@
+# Automation
+
+There are some Organization-wide secrets available for GitHub Automation of various workflows.
+
+## Personal Access Tokens
+
+Automation currently depends on the availability of two GitHub Personal Access Tokens (PATs).
+
+PATs have an expiry and the secret will have to be updated periodically by admins. An admin will create
+a PAT on their account, requesting access to the organization. See [the docs](https://docs.github.com/en/organizations/managing-programmatic-access-to-your-organization/managing-requests-for-personal-access-tokens-in-your-organization).
+
+### Release PAT
+
+The release PAT requires permissions:
+- Read access to actions variables, metadata, and secrets for all current and future repositories owned by this
+  organization
+- Read and Write access to code, issues, and pull requests for all current and future repositories owned by this
+  organization
+
+### Sonar PAT
+
+The sonar PAT requires permissions:
+- Read access to metadata for all current and future repositories owned by this organization
+- Read and Write access to code and pull requests for all current and future repositories owned by your organization
+
+# Actions secrets and variables
+
+The Organization sets some Actions secrets so they are available to all repositories
+
+| Name                 | Description                                                         | Example       |
+|----------------------|---------------------------------------------------------------------|---------------|
+| DEPENDENCY_TOKEN     | ???                                                                 | secret :lock: |
```

Comment:
this looks to be used by console to merge dependabot PRs, is it pointing at the Release PAT?

Reply:
No, this is a separate token with less access than the release PAT.

Reply:
It has read access to metadata, and read/write to code + pull requests.

Reply:
thanks I was mixed up I had it in my mind the PAT had sonar in it's name but it's called `streamshub-dependency-token`, d'oh!

Reply:
updated the doc